src/apprt/gtk/Window.zig

```diff
@@ -661,6 +661,16 @@ pub fn toggleWindowDecorations(self: *Window) void {
     };
 }
 
+/// Toggle the background opacity for this window.
+pub fn toggleBackgroundOpacity(self: *Window) !f64 {
+    if (self.app.config.@"background-opacity" >= 1) return self.app.config.@"background-opacity";
```

Comment:
What's this for?

Reply:
Well if it is global state then it shouldn't be stored per-window. Should be part of the App's DerivedConfig instead.

Reply:
okay cool that sounds good. I wasn't sure if I should be modifying that for this logic but that makes sense, I can update that

Reply:
I might just be blind but I don't see a Derived Config at the app level, only the package config that you can set from modifying your ghostty config